ui/src/doc-inline/it/graphs_avg_duration_1_line_queueDuration.md

```diff
@@ -0,0 +1,15 @@
+# Durata Chiamata
+
+*Durata della Chiamata* per le code selezionate nell'orario lavorativo impostato.
```

Comment:
```suggestion
```
This line is probably not needed, next sentence says the same concepts and adds more details